### DISCUSSION TITLE
Posting Media files to Twitter

### DISCUSSION THREAD

**Discussion by caldvdsf** [elizaOS/eliza#1216] [General]
I'm wondering if anyone has had any with the posting of media files to twitter? so the two ways I want this to work is: 1. A folder that is used to keep files I want the bot to rotate through on a cadence (like text tweet, text tweet, text+image (generated on the fly), text tweet, etc. and
2. as a response to a comment, like if they post a text tweet and someone replies generate an image of... it will return the image with some text response. Does it have the capability to do this?

**Comment by 0xTheDot** [comment]
We mindjourney API ? :))

**Comment by luduvigo** [comment]
Question: don't you think it's too custom for the eliza repo?

**Comment by Lyonsupernova** [comment]
https://github.com/elizaOS/eliza/blob/76d4f42e5763f638095d1179b2e88afc03fb8896/packages/client-twitter/src/utils.ts#L186C1-L186C71

just glance through this line, if you need to attach the picture to the media try add pic uri to the attachment of content.

**Reply by rubenmarcus** [reply]
tried this on my character.ts file on messageExamples , still didnt posti mages on replies or posts

**Comment by themeshri** [comment]
Have you managed to solve this? @caldvdsf
